website/content/docs/release-notes/1.13.0.mdx

```diff
@@ -103,11 +103,17 @@ Some of these enhancements and changes in this release include the following:
 
 ## Known issues
 
-There are no known issues documented for this release.
+When you use Vault 1.13.0 without a TLS certificate on the TCP listener through which you access the Vault WebUI you may receive an error like the one below when performing operations within the Vault WebUI. This is an unintended behaviour that is resolved in the forthcoming Vault 1.13.1 release.
+
+```
+Q.randomUUID is not a function
+```
+
+-> Refer to this [Knowledge Base article](https://support.hashicorp.com/hc/en-us/articles/14512496697875) for more details and a workaround.
```

Comment:
@rowansmithhc Thanks for this PR.  I would like to suggest the following edits.

```suggestion
When Vault is configured without a TLS certificate on the TCP listener, the Vault UI may throw an error that blocks you from performing operational tasks. 

The error message: `Q.randomUUID is not a function`

<Note>

Refer to this [Knowledge Base article](https://support.hashicorp.com/hc/en-us/articles/14512496697875) for more details and a workaround.

</Note>

The fix for this UI issue is coming in the Vault 1.13.1 release.
```



This will look like:

![image](https://user-images.githubusercontent.com/7660718/226494031-b809a015-33ab-453b-9d87-f3fd4238bde0.png)